views/result.erb

```diff
@@ -0,0 +1,23 @@
+<%= @name %>: <%= @user_choice %>
+<br>
+Computer: <%= @cpu_choice %>
+<br>
+  <% if @user_choice == "rock" && @cpu_choice == "scissors" %>
+    You win! :)
+  <% elsif @user_choice == "scissors" && @cpu_choice == "paper" %>
+    You win! :)
+  <% elsif @user_choice == "paper" && @cpu_choice == "rock" %>
+    You win! :)
+  <% elsif @user_choice == @cpu_choice %>
+    It's a draw! :/
+  <% else %>
+    You lose! :(
+  <% end %>
+<br>
```

Comment:
I like how you used else to cover the computer winning the game, saves writing double the amount of code! Wish I'd thought of doing that for mine